### DISCUSSION TITLE
Is it possible to use this as a reusable workflow?

### DISCUSSION THREAD

**Discussion by boojamya** [leonsteinhaeuser/project-beta-automations#56] [Q&A]
Hello!
I'm hoping to use this as a reusable workflow using the `workflow_call:` feature in Github actions.

For example, I recently needed to update from project-beta-automations version from v1.2.1 to v2.0.1.
Since this is used in several repos, it needed to be changed multiple times. It would be nice to centralize this.

Even though the repos are under the same Org, it still is having permission issues. 

I did try passing the `resource_node_id` as an input with no luck.

**Comment by leonsteinhaeuser** [comment]
@boojamya It should be possible as long as you provide the necessary credentials and values. 

I personally never tested it, but the action doesn't depend on things related to the runtime environment (GitHub action).

**Reply by leonsteinhaeuser** [reply]
@boojamya have you tried to enable the debug option and check what actually happens? If you could provide such a debug log and a simple example, I could check it too.

**Comment by aptalca** [comment]
Yes, we are doing it on a pretty large scale as we maintain hundreds of repos.
Here's a simple calling workflow that exists in all of our repos: https://github.com/linuxserver/github-workflows/blob/v1/.github/workflows/call-issue-pr-tracker.yml
And here's a reusable workflow that contains all the logic: https://github.com/linuxserver/github-workflows/blob/v1/.github/workflows/call-issue-pr-tracker.yml
Here's our v2 project board: https://github.com/orgs/linuxserver/projects/8

The reason we went that route is because we didn't want to have to update hundreds of repos every time we wanted to make a change in the columns or the logic

We did have to use a PAT even though everything is within the same org. The PAT is stored as an org secret.
